examples/pom.xml

```diff
@@ -11,13 +11,27 @@
         <java.version>1.8</java.version>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
+        <rapid-java-sdk.sdk.version>LATEST</rapid-java-sdk.sdk.version>
     </properties>
 
+    <repositories>
```

Comment:
Will this action only work with snapshot releases? How did it work before without this part added?

Reply:
Yes. It did not work on my local machine before. However, at the development time of creating these examples, it was enabled in the `settings.xml` file on the local machine of the developer.